.ci/update-beats.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: Bump beats
+title: Bump beats
+
+scms:
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: reakaleek
+      repository: apm-server
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: '{{ requiredEnv "BRANCH_NAME" }}'
+
+actions:
+  apm-server:
+    kind: github/pullrequest
+    scmid: githubConfig
+    spec:
+      automerge: false
+      labels:
+        - automation
+        - backport-skip
+
+sources:
+  beats:
+    kind: shell
+    spec:
+      command: make get-latest-beats-version
+      environments:
+          - name: PATH
+
+conditions:
+  beats-version-check:
+    name: Check beats version
+    kind: shell
+    sourceid: beats
+    spec:
+      command: sh .ci/scripts/check-beats-version.sh
```

Comment:
I've seen sometimes that `sh` might be pointing to `dash`. What do you think by adding the shebang to the script `.ci/scripts/check-beats-version.sh` in addition to granting execution permissions to the script so this command could be just `command: .ci/scripts/check-beats-version.sh`

Reply:
https://github.com/elastic/apm-server/pull/10179/commits/b15fc22634e09bfd9e56094b9060e8303405f59a